### ISSUE TITLE
[ENH] Standardize Parameters and Returns

### ISSUE THREAD

**Issue by pbulsink** [SCasanova/f1dataR#49]
We should standardize parameter names and return types across the package.

This will end up being a breaking change unfortunately, but would likely be required if you ever plan to submit to CRAN.

For example of parameters, the two functions below should have the same input parameter names:
```
pitstops <- load_pitstops(season = season, race = race)
quali <- load_quali(season = season, round = race)
```

For example of returns, `load_session_laps()` returns a data.frame, while the Ergast calling functions return tibbles.

Finally, functions don't have a constant column name style: `load_results()` and others return columns with both camelCaseNames and snake_case_names. The FastF1 functions return columns with CamelCase (leading capital letter). Having one explicit style would be best.

Code styling could follow https://style.tidyverse.org/index.html, where it suggests names follow snake_case. 

Before I start looking at these changes I'd love your feedback, @SCasanova!

**Comment by SCasanova** [comment]
I can take a look at this and other issues in the coming week. Sorry about the lack of comunication

**Comment by SCasanova** [comment]
Also, I agree with the overall snake case, and probably best to have all outputs be data frames. I'll get on top of that

**Comment by pbulsink** [comment]
Tibbles are nice.

I think as you progress through changing function parameters (and function
names!) use the suggestions from the lifecycle package:
https://cran.r-project.org/web/packages/lifecycle/vignettes/communicate.html

Let me know what functions you want a hand with. I think once a standard is
set (i.e. input parameter/arguement names, output style, rename functions,
etc) I can help move through the list.

**Comment by SCasanova** [comment]
As a standard, I thought about changing all "race" parameters to "round" and then using ´janitor::clean_names()´ to ger all column names into snake case in a consistent manner. If you agree with tibbles I like that as the standard output as well. I haven't given the actual function names much though

**Comment by SCasanova** [comment]
Btw the package is currently broken on my machine because of M1 silicon issues so I can't do any testing at the moment

**Comment by pbulsink** [comment]
> Btw the package is currently broken on my machine because of M1 silicon issues so I can't do any testing at the moment

If you code it up in a branch and do a pull request, then each commit you add to that branch after you make the pull request will reinitialize testing on the github servers. Not as easy as doing it locally, but close.

**Comment by SCasanova** [comment]
@pbulsink let me know if you think more action is needed here when you have time. Thanks!

**Comment by pbulsink** [comment]
I'll take a look and get back to you here! 

I'm in the midst of using the data produced by the package for some predictive modelling, so regretting suggesting this change mid-season but I think it's for the best.

**Comment by pbulsink** [comment]
Likely with both all of these changes and vignette updates (#59) we'll need a version change to v1.0. Normal version numbering would suggest that breaking changes like this requires a new major vX.0.0 version bump, but we can still get away with just going to 0.5.0 as a feature of the 'beta' status implied by v0 if you'd rather?

**Comment by SCasanova** [comment]
Closed by #63, #64, #65, #69
